src/xFactor/base/Application.java

```diff
@@ -0,0 +1,257 @@
+		addJudges();
+		for (int i = 0; i < number; i++) {
+			judges.add(allJudges.get(i));
+		}
+		judgeDao = new JudgeDao();
+		for (int i = 0; i < judges.size(); i++) {
+			judgeDao.save(judges.get(i));
+		}
+	}
+
+	public ArrayList<Judge> getJudges() {
+		return judges;
+	}
+
+	public void setJudges(ArrayList<Judge> judges) {
+		this.judges = judges;
+	}
+
+	public int getJudgesSize() {
+		return judges.size();
+	}
+
+	public void giveSpecialVote(int id) {
+		if (id > 0 && id < judges.size()) {
+			for (Judge judge : judges) {
+				if (judge.getJudgeId() == id) {
+					judge.setSpecial(true);
+					judgeDao.update(judge);
+				}
+			}
+		}
+	}
+
+	public void printSpecialJudge() {
+		for (Judge judge : judges) {
+			if (judge.isSpecial()) {
+				System.out.println(judge.getName());
+				break;
+			}
+		}
+	}
+
+	public void printJudges() {
+		for (Judge judge : judges) {
+			System.out.println(judge);
+		}
+	}
+
+	public void createParticipants(int number) {
+		participants = new ArrayList<Participant>();
+		addParticipants();
+		for (int i = 0; i < number; i++) {
+			participants.add(allParticipants.get(i));
+		}
+		participantDao = new ParticipantDao();
+		for (int i = 0; i < participants.size(); i++) {
+			participantDao.save(participants.get(i));
+		}
+	}
+
+	public void createStage(int stageNum, int max) {
+		Stage stage = new Stage(stageNum, max);
+		allStages.add(stage);
+		initializeStage(stageNum, max, stage);
+		stageDao = new StageDao();
+		stageDao.save(stage);
+	}
+
+	public void printParticipantsInStage(Stage stage) {
+		System.out.println("Participants in stage " + stage.getStageNumber() + ":\n" + stage.getParticipantsInStage());
+	}
+
+	public void addQualitiesForParticipants() {
+		fillQualities();
+		for (Participant participant : participants) {
+			int i = random.nextInt(qualities.size() - 1);
+			participant.addQuality(qualities.get(i));
+			participantDao.update(participant);
+		}
+	}
+
+	public void printParticipantsInfo() {
+		for (Participant participant : participants) {
+			participant.printInfo();
+		}
+	}
+
+	public void printJudgesInfo() {
+		for (Judge judge : judges) {
+			personService.printInfo(judge);
+		}
+	}
+
+	public void addJudgesFavourites(Stage stage) {
+		judgeFavouriteDao = new JudgeFavouriteDao();
+		for (Judge judge : judges) {
+			for (int i = 1; i <= judge.getMaxFavourites(); i++) {
+				int rand = random.nextInt(participants.size());
+				Participant participant = participants.get(rand);
+				stage.addFavourite(participant, judge);
+				stageDao.update(stage);
+				judgeFavouriteDao.save(new JudgeFavourite(participant, judge));
+			}
+		}
+	}
+
+	public void printFavourites(Stage stage) {
+		System.out.println("favourites: " + stage.getJudgeFavourites());
+	}
+
+	public void judgesVote(Stage stage) {
+		voteDao = new VoteDao();
+		for (Participant participant : stage.getParticipantsInStage()) {
+			for (Judge judge : judges) {
+				Vote vote = stage.vote(judge, participant, random.nextBoolean());

```

Comment:
you should not create the vote with the stage item, better create it here and add it separately in the stage